src/resources/app/components/AnnouncementContent.vue

```diff
@@ -7,84 +7,57 @@
           <div id="author">
             {{ name }}
           </div>
-          <div
-            id="date"
-            class="text-sm text-textGray">
+          <div id="date" class="text-sm text-textGray">
             {{ announcementDate }}
           </div>
         </div>
       </div>
       <v-menu
         id="menu"
-        v-if="getUserId === this.$props.authorId || getUserName === this.$props.name"
+        v-if="
+          getUserId === this.$props.authorId || getUserName === this.$props.name
```

Comment:
since it's quite long, consider refactoring this as a new function or computed, e.g. `isCurrentUserTheAuthor` or something along those lines

Reply:
I think, that's my job to refactor it 😅😅😅. Thank you Aldi 🙏🙏

Reply:
done! thanks for noticing di